src/crimson/os/seastore/extent_placement_manager.cc

```diff
@@ -225,6 +225,7 @@ void ExtentPlacementManager::init(
 	    rb_cleaner));
     // TODO: implement eviction in RBCleaner and introduce further writers
     data_writers_by_gen[generation_to_writer(OOL_GENERATION)] = writer_refs.back().get();
+    data_writers_by_gen[generation_to_writer(MIN_REWRITE_GENERATION)] = writer_refs.back().get();
```

Comment:
I initially believed that `get_writer()` should obtain the writer from REWRITE_GENERATION since it operates during the rewrite phase. I replaced this with OOL_GENERATION.

Reply:
One of the original goal of `MIN_REWRITE_GENERATION` is to isolate the segments of (io-)ool and rewritten(-ool) through different writers, so that reclaiming segments can be more efficient.

RBM doesn't seem to need this, all are randomly written as ool through the same writer without tiering.

Reply:
OK. I understand. Already removed.